### PR TITLE
Add 9482 to module scripting

### PR DESCRIPTION
**Pull request by oscarfonloz**
# This PR requires the branch ["users/lynn/9225"](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/pull/112/files) to go in first, since it uses those changes

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Add the 9482 module to the scripting API

### Why should this Pull Request be merged?

Task 1131948

### What testing has been done?

Added automated test for this module. Fixed existing tests that had hard coded constants and ensured the other scripting modules still passed all its tests.

### CODE REVIEW

**Comment by rtzoeller** [comment]
- [ ] `Test Counter Driven Output.vi`, `Test Digital Output.vi`, and `Test PWM Output.vi` are now visually very dense, which makes them hard to follow. It would be helpful if the "default config" logic was separated a bit more from the "random config" logic, i.e. add horizontal whitespace rather than locating it under the other section.
- [ ] `Convert Module Model to Module Class.vi` needs to be updated to include this class. Please update this VI and re-order the cases to be sorted.

**Comment by oscarfonloz** [comment]
> * [ ]  `Test Counter Driven Output.vi`, `Test Digital Output.vi`, and `Test PWM Output.vi` are now visually very dense, which makes them hard to follow. It would be helpful if the "default config" logic was separated a bit more from the "random config" logic, i.e. add horizontal whitespace rather than locating it under the other section.
> * [ ]  `Convert Module Model to Module Class.vi` needs to be updated to include this class. Please update this VI and re-order the cases to be sorted.

Fixed both, @rtzoeller :)

**Review by buckd — COMMENTED** [review]
- [ ] `Convert Module Model to Module Class.vi` has the wrong case label for this class.
image
- [ ] `NI 9482.lvclass` and its member VIs should be set to separate source.
- [ ] Group all the modules together in the scripting library. It just makes it easier to find things as the module count keeps growing.
image

**Comment by oscarfonloz** [comment]
>  Convert Module Model to Module Class.vi has the wrong case label for this class.
> image
>  NI 9482.lvclass and its member VIs should be set to separate source.
>  Group all the modules together in the scripting library. It just makes it easier to find things as the module count keeps growing.

Fixed all 3 comments, thank you @buckd!!

**Review by buckd — APPROVED** [review]
Waiting on @rtzoeller to do a final pass before merging.
